### PR TITLE
[Chore] add health check endpoint

### PR DESCRIPTION
**Pull request by nodox**
We want a way to moniter cronut in Pingdom. Thus a health
check endpoint is created to allow the service to see if our
clock work job has fired.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -9,69 +9,78 @@
 # from scratch. The latter is a flawed and unsustainable approach (the more migrations
 # you'll amass, the slower it'll run and the greater likelihood for issues).
 #
-# It's strongly recommended to check this file into your version control system.
+# It's strongly recommended that you check this file into your version control system.
```
Comment (irubnich) [inline db/schema.rb]: So, from offline talking, these changes stem from the fact that the Rails version was bumped but this file wasn't regenerated. The changes are pretty minimal so I'm cool with including this in the PR.

File: app/models/health_check.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+class HealthCheck < ActiveRecord::Base
+  THRESHOLD = 2.minutes
+
+  def self.get_time_of_last_fired_job
```
Comment (irubnich) [inline app/models/health_check.rb]: This method isn't used anywhere

**Comment by werkshy** [comment]
This is neat, I like the approach of storing the clock run in the DB and checking the DB for it - two birds, one stone.

**Comment by nodox** [comment]
You can thank @pjambet for the idea!

**Pull request merged**